server/base/src/main/java/org/apache/accumulo/server/client/ClientServiceHandler.java

```diff
@@ -447,6 +460,83 @@ public List<TDiskUsage> getDiskUsage(Set<String> tables, TCredentials credential
     }
   }
 
+  @Override
+  public List<FateTransaction> executeAdminOperation(TInfo tInfo, TCredentials credentials,
+      AdminOperation op, List<String> txids, List<String> filterStatuses)
+      throws ThriftSecurityException, TException {
+    try {
+      authenticate(tInfo, credentials);
```

Comment:
In addition to authenticating, we probably want to restrict this to users with admin permissions, right?

Reply:
What do you mean by "admin permissions" here? It looks like this checks that the credentials contain `SYSTEM` permissions.

Reply:
> What do you mean by "admin permissions" here? It looks like this checks that the credentials contain `SYSTEM` permissions.

What line do you see that?

Reply:
I mean `SystemPermission.SYSTEM`, or `security.canPerformSystemActions(credentials)`